index.html

```diff
@@ -1,11 +1,43 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Lesson 1</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TripHouse_Site</title>
+    <link href="style.css" rel="stylesheet">
 </head>
 <body>
-<h1>Моя первая страница</h1>
-<a href="About/index.html">Об авторе</a>
+<div class="advantages" id="advantages">
```

Comment:
**Semantic recommendation** 
It's better to use `<section> ` here.
The` <section>` tag defines a section in a document.
![image](https://user-images.githubusercontent.com/25488411/156405622-de79942f-de2a-4d42-8651-6de5a089d580.png)